modules/nodes-secondary-scheduler-pod-console.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-pod-console_{context}"]
+= Scheduling a pod using the secondary scheduler
+
+To schedule a pod using the secondary scheduler, set the `schedulerName` field in the pod definition.
+
+.Prerequisities
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* The {secondary-scheduler-operator-full} is installed.
+* A secondary scheduler is configured.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Navigate to *Workloads* -> *Pods*.
+. Click *Create Pod*.
+. In the YAML editor, enter the desired pod configuration and add the `schedulerName` field:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  namespace: default
+spec:
+  containers:
+    - name: nginx
+      image: nginx:1.14.2
+      ports:
+        - containerPort: 80
+  schedulerName: secondary-scheduler <1>
+----
+<1> The `schedulerName` must match the name that is defined in the config map when you configured the secondary scheduler.
```

Comment:
Suggestion to specify what kind of object the literal value is, I think this comes from the supplemental guidelines for [object references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) (I haven't seen it elsewhere)
```suggestion
<1> The `schedulerName` parameter must match the name that is defined in the config map when you configured the secondary scheduler.
```